manifests/params.pp

```diff
@@ -10,7 +10,11 @@
   }
 
   $ensure                      = 'present'
-  $version                     = 'system'
+  if $facts['os']['family'] == 'RedHat' and $facts['os']['release']['major'] == '8' {
+    $version                   = '3'
```

Comment:
feels like we should change `$version` to `'3'` as default for all platforms. We set that in the acceptance tests and don't test anything else. (should be changed in another PR)

Reply:
I think the exception should be EL7 because it feels wrong to do something different than what ships with a standard install of the OS.  I think the logic could just be if EL7 , version=2 else version=3.

Reply:
I will try to do that in another PR